### PR TITLE
[ws-daemon] Create missing LVM device and special files

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Without LVM devices and special files (symlinks), we cannot enforce disk quotas.
In installations without LVM this is a NOOP.

## How to test
- Create an ephemeral cluster using LVM
- Open a workspace
- Run `fallocate -l 100G /workspace/should-fail`
- Should return with an error
 
## Release Notes
```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by roboquat** [comment]
@aledbf: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/ws-daemon/cmd/run.go [hunk 1]
```diff
@@ -142,3 +145,13 @@ func grpcProbe(cfg baseserver.ServerConfiguration) func() error {
 		return fmt.Errorf("grpc service not ready")
 	}
 }
+
+// createLVMDevices creates LVM logical volume special files missing when we run inside a container.
+// Without this devices we cannot enforce disk quotas. In installations without LVM this is a NOOP.
+func createLVMDevices() {
+	cmd := exec.Command("/usr/sbin/vgmknodes")
```
Comment (utam0k) [inline components/ws-daemon/cmd/run.go:152]: Is it safe to be called twice? I'm afraid of when restarting `ws-daemon`

**Review by utam0k — APPROVED** [review]
/hold
Wait until the comment is responded to. The code itself seems fine.

File: components/ws-daemon/cmd/run.go (hunk 1 above)
Comment (aledbf) [reply components/ws-daemon/cmd/run.go:152]: Yes.

Comment (utam0k) [reply components/ws-daemon/cmd/run.go:152]: @aledbf Thanks for your insight :pray:

**Pull request merged**